AUTHORS.txt

```diff
@@ -18,6 +18,7 @@ Maurizio M. Gavioli <mmg@vistamaresoft.com>
 Nicolas Semrau <nhs@openmailbox.org>
 Philipp H. Poll <gillian@linuxlibertine.org>
 Stefan Peev <context.bg@gmail.com>
+Felix Kopecky <felix.kopecky@langsci-press.org>
```

Comment:
This list is alphabetically ordered, so please move your line after D and before G.